### PR TITLE
Fix the issue with wine and update deps while I'm at it

### PR DESCRIPTION
**Pull request by jessb321**
This should (hopefully) be the last PR I have to make, since hopefully the addition of wine should fix this for a while.  
Really do like this project though. (You can probably tell 😉 )

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,16 @@
-sudo: false
+sudo: required
+dist: trusty
 language: node_js
+before_install:
+- sudo dpkg --add-architecture i386
+- sudo apt-get update
+- sudo apt-get install -y wine
+
+install:
```
Comment (azu) [inline .travis.yml]: Does replace to`before_script:`?
https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI

**Review by azu — APPROVED** [review]
Cool!

File: .travis.yml (hunk 1 above)
Comment (jessb321) [reply .travis.yml]: Yeah, I can put the xvfb in the before_script.

**Comment by azu** [comment]
@willyb321 Thank you always!
Release v4.0.0 https://github.com/azu/electron-zip-packager/releases/tag/4.0.0

P.S. I've added @willyb321 to collaborators(GitHub and npm). Thanks!

**Comment by jessb321** [comment]
Thanks!!

**Pull request merged**